test/e2e/certificates_test.go

```diff
@@ -354,6 +355,65 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("ACME Certificate dns-01 challenge using ambient credentials should obtain a valid LetsEncrypt certificate in GCP mint/passthrough clusters-65035", func() {
+		It("should obtain a valid LetsEncrypt certificate", func() {
+
+			By("getting Infrastructure object")
+			infra, err := configClient.Infrastructures().Get(ctx, "cluster", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			if infra.Status.PlatformStatus.Type != configv1.GCPPlatformType {
+				Skip("Skip this case for current plaform, it's not gcp.")
+			}
+
+			By("Check cloud credential in cluster")
```

Comment:
- Sometime it is initial lower case do-ing format `By("getting`, `By("creating`, sometimes it is initial upper case verb format `By("Check`, `By("Configure`. This seems to make inconsistency in RH public repo. It is better to unify format.
- Same for `("Skip ` and `"Skipping `.

Reply:
Updated.